authentication/views.py

```diff
@@ -44,7 +46,7 @@ def post(self, request):
         serializer = self.serializer_class(data=user)
         serializer.is_valid(raise_exception=True)
 
-        return Response(serializer.data, status=status.HTTP_200_OK)
+        return Response({'token': serializer.data.get('token')}, status=status.HTTP_200_OK)
```

Comment:
**Why did you change this login response?**

Reply:
The user does not pass in the token as they fill in the credentials when registering, so i changed the login reposnse to be able to return the user a token as well.

Reply:
I, have fixed the commit message as requested.

Reply:
I don't think this line requires your changes because your task is about registration but not login. So no need to touch the login implementation!!!

Reply:
Alright let me fix this ASAP

Reply:
Removed the token form the login response as requested